pkg/server/server.go

```diff
@@ -26,14 +26,15 @@ import (
 	"sync"
 	"time"
 
+	runpprof "runtime/pprof"
```

Comment:
Seems like this should actually be between "io" and "strconv".

Reply:
oh yeah, I think my editor did this automatically...